linux/Platform.c

```diff
@@ -325,15 +325,17 @@ double Platform_setCPUValues(Meter* this, unsigned int cpu) {
       v[CPU_METER_IOWAIT]  = cpuData->ioWaitPeriod / total * 100.0;
       this->curItems = 8;
       if (this->pl->settings->accountGuestInCPUMeter) {
-         percent = v[0] + v[1] + v[2] + v[3] + v[4] + v[5] + v[6];
+         percent = v[CPU_METER_NICE] + v[CPU_METER_NORMAL] + v[CPU_METER_KERNEL] + \
+            v[CPU_METER_IRQ] + v[CPU_METER_SOFTIRQ] + v[CPU_METER_STEAL] + v[CPU_METER_GUEST];
       } else {
-         percent = v[0] + v[1] + v[2] + v[3] + v[4];
+         percent = v[CPU_METER_NICE] + v[CPU_METER_NORMAL] + v[CPU_METER_KERNEL] + \
+            v[CPU_METER_IRQ] + v[CPU_METER_SOFTIRQ];
```

Comment:
No word-wrapping long lines. Also no backslash for line continuation outside macros.